### PR TITLE
[tech][templates] moving the scaffold project's asset loader outside of the defs

### PR DESCRIPTION
**Pull request by tacastillo**
## Summary & Motivation

Separating the `load_assets_from_module` call from it getting passed as an argument into `Definitions`. Mainly for readability/consistency since the tutorial/guides and other things all define things outside of the `Definitions`

### CODE REVIEW

File: python_modules/dagster/dagster/_generate/templates/CODE_LOCATION_NAME_PLACEHOLDER/CODE_LOCATION_NAME_PLACEHOLDER/__init__.py [hunk 1]
```diff
@@ -2,4 +2,8 @@
 
 from . import assets
 
-defs = Definitions(assets=load_assets_from_modules([assets]))
+all_assets = load_assets_from_modules([assets])
+
+defs = Definitions(
+    assets=all_assets
```
Comment (sryza) [inline python_modules/dagster/dagster/_generate/templates/CODE_LOCATION_NAME_PLACEHOLDER/CODE_LOCATION_NAME_PLACEHOLDER/__init__.py:8]: I think you need to add a comma at the end of this line or the auto-formatter will try to collapse it onto a single line.

Comment (tacastillo) [reply python_modules/dagster/dagster/_generate/templates/CODE_LOCATION_NAME_PLACEHOLDER/CODE_LOCATION_NAME_PLACEHOLDER/__init__.py]: Odd, my precommit hook didn't kick in. updated and ran `make black` to be sure! ty!

**Comment by sryza** [comment]
This LGTM. @yuhan - do you want a chance to look before I approve?

**Review by yuhan — APPROVED** [review]
lg2m!

**Pull request merged**